GOVERNANCE.md

```diff
@@ -33,12 +35,13 @@ A PR should be created with all relevant changes to update the repository for th
 It should apply the following actions:
 
 1. Change the "Unreleased version" heading in the CHANGELOG.md file
```

Comment:
@Xymph any thoughts about renaming this to "Upcoming version" or "Next version", as I suggested [here](https://github.com/hamstar/Wikimate/pull/102#discussion_r662137004)?

Reply:
"Upcoming version" works for me as well.

Reply:
Done in 1a9ae641cced375474d378f10a0654d562575623 :)